### PR TITLE
couple suggestions

### PR DESCRIPTION
**Pull request by witmicko**
@jstaffor @austincunningham 

couple changes Id suggest before merging parent PR.

### CODE REVIEW

File: docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc [hunk 1]
```diff
@@ -2,11 +2,11 @@
 = Introducing {WFM-RC-NameShort} Security Features
 
 {WFM-RC-NameShort} Security is built around a *Security Interface* which abstracts itself from all {WFM-RC-NameShort} security libraries.
-{WFM-RC-NameShort} allows you to build a custom security solution for integration with {WFM-RC-NameShort}.
-{WFM-RC-NameShort} also provides two security implementations out-of-the-box and are called link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
+{WFM-RC-NameShort} allows you to integrate custom security solution.
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc]: integrate "a" custom security solution

File: docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
+{WFM-RC-NameShort} also provides two out-of-the-box security implementations: link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc]: +1 for shortening the text. Best to drop the " : " and use English where possible, " security implementations called ..."

File: docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
 
-*Keycloak* and *Passport.js* both provide authentication and authorization on the server side.
-*Keycloak* and *Passport.js* implementations can replace each other without breaking the internal api belonging to all {WFM-RC-NameShort} module's and also any of the reference applications.
+*Keycloak* and *Passport.js* both provide server side authentication and authorization.
+*Keycloak* and *Passport.js* implementations can replace each other without breaking the internal {WFM-RC-NameShort} api, modules and also any of the reference applications.
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc:9]: This change has a different meaning than the previous text.

If it still holds true from a technical POV - I'm a +1 as it reads better.

File: docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc [hunk 4]
```diff
@@ -22,15 +22,15 @@ NOTE: All link:https://expressjs.com/[express] routes outside of WFM modules mus
 //Pasport.js
 == Introducing Passport.js
 
-*Passport.js* is a lightweight authentication middleware solution for Node.js and Can be bundled into existing Node.js server.
+*Passport.js* is a lightweight authentication middleware solution for Node.js and can be bundled into existing Node.js server.
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc]: bundled into "an" existing

File: docs/workforce-management-framework/topics/con-passportauth-module.adoc [hunk 5]
```diff
@@ -1,9 +1,11 @@
 [id='con-passportauth-module-{chapter}']
 = Introducing the {WFM-RC-NameShort} PassportAuth Module
 
-The link:{WFM-RC-CoreTreeURL}{WFM-RC-Branch}/cloud/passportauth[PassportAuth] module is {WFM-RC-NameShort}'s implementation of *Passport.js* and is the default authentication and authorization module for {WFM-RC-NameShort}.
-*PassportAuth* provides pluggable authentication using link:http://passportjs.org/docs/configure[strategies].
-*PassportAuth* offers a comprehensive set of strategies that support authentication using a username and password, Facebook logon, Twitter logon, and more.
+The link:{WFM-RC-CoreTreeURL}{WFM-RC-Branch}/cloud/passportauth[PassportAuth] module is {WFM-RC-NameShort}'s implementation of *Passport.js*.
+It is the default authentication and authorization module for {WFM-RC-NameShort}.
+
+*passport.js* provides pluggable authentication using link:http://passportjs.org/docs/configure[strategies].
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/con-passportauth-module.adoc]: This has changed from *PassportAuth* to *passport.js*. This doesn't make sense to me as this module is introducing the PassportAuth Module. This should be  *PassportAuth* or else we remove this line and the line below as I do not want to confused the reader (as I was).

File: docs/workforce-management-framework/topics/pro-working-with-passportauth.adoc [hunk 6]
```diff
@@ -35,11 +35,9 @@ You must have completed:
 //<***TODO***>
 //----
 == Configuration of PassportAuth module
-
-As the PassportAuth module is the default provider of authentication and authorisation, it is already integrated 
-with the Demo applications. It is an out of the box security solution for default integration using static 
-JSON files as user source. The sample users and access roles are defined in a JSON file for Demo purposes in the 
-link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/wfm-user/users.json[Demo Data] To use different source please follow guide bellow. RainCatcher PassportAuth is a lightweight solution 
+The module is an out of the box security solution for default integration. It is using JSON files
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/pro-working-with-passportauth.adoc]: "The PassportAuth is the out of the box security solution for the default integration of {WFM-RC-NameShort}."
What is meant by "default integration"?

"PassportAuth uses JSON files that contain user data."

File: docs/workforce-management-framework/topics/pro-working-with-passportauth.adoc [hunk 7]
```diff
@@ -35,11 +35,9 @@ You must have completed:
...
+as user data source. The sample users and access roles are defined in the JSON file for Demo purposes in the
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/pro-working-with-passportauth.adoc]: "This user data contains sample users and access roles for demo purposes..."

File: docs/workforce-management-framework/topics/pro-working-with-passportauth.adoc [hunk 8]
```diff
@@ -35,11 +35,9 @@ You must have completed:
...
+link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/wfm-user/users.json[Demo Data]. To use different source please follow guide bellow. RainCatcher PassportAuth is a lightweight solution
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/pro-working-with-passportauth.adoc]: Replace:
"To use different source please follow guide bellow. RainCatcher PassportAuth is a lightweight solution"

with

"The following procedure describes how to use a different user data source." ("RainCatcher PassportAuth is a lightweight solution that offers all the functionality required to get started." was removed completely as it's of no benefit to the reader).

File: docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc [hunk 9]
```diff
@@ -19,9 +19,9 @@ For more information, see link:https://github.com/expressjs/session[express-sess
 
 [id='defining-access-control-{chapter}']
 == Defining Access Control
-*PassportAuth* provides role based access control and _routes_ are protected by these roles.
-_authService_ is an an implementation of link:../../../raincatcher-docs/{WFM-RC-Api-Version}{WFM-RC-Api-Passport-Auth}[PassportAuth].
-For more information, see the following code snippet:
+_Routes_ are protected by *PassportAuth* provided role based access control.
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc]: by *PassportAuth* "and provide" role based access control.

File: docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc [hunk 10]
```diff
@@ -30,19 +30,17 @@ app.get('/testAdminEndpoint', authService.protect('admin'), (req: express.Reques
...
-NOTE: Demo roles are defined as an array in link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/wfm-user/users.json[user.json].
+NOTE: Demo roles are defined in link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/wfm-user/users.json[user.json].
 
 [id='defining-password-storage-{chapter}']
 == Defining Password Storage
-The link:../../../api/{WFM-RC-Api-Version}{WFM-RC-Api-User-Repository}#getuserbylogin[User Repository Interface] 
-is used to store *PassportAuth* user data.
-To point *PassportAuth* to a datasource, you must implement the User Repository Interface.
+The *PassportAuth* user data is stored in link:../../../api/{WFM-RC-Api-Version}{WFM-RC-Api-User-Repository}#getuserbylogin[User Repository Interface].
+User Repository Interface must be implemented in order to point *PassportAuth* to a datasource,
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc]: "Make sure that the _User Repository Interface_ is implemented..."

I only learned this sentence structure this week as a result of a peer review - the docs style guide states that we should avoid using the word "must" and "have to".

File: docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc [hunk 11]
```diff
@@ -30,19 +30,17 @@ app.get('/testAdminEndpoint', authService.protect('admin'), (req: express.Reques
...
 
 [id='passportauth-authentication-{chapter}']
 == PassportAuth Authentication
-{WFM-RC-NameShort} uses the Default Strategy for username/password authentication, and authentication happens on 
-the cloud server. The *PassportAuth* authentication service uses a default custom strategy that is based on the 
-local *Passport.js* strategy.The *PassportAuth* local strategy is database agnostic.
+{WFM-RC-NameShort} uses username/password as the default authentication strategy, credentials are authenticated on the cloud server.
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc]: uses "a username and password combination" as the..


authentication strategy "and the" credentials are authenticated on the cloud server.

File: docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc [hunk 12]
```diff
@@ -30,19 +30,17 @@ app.get('/testAdminEndpoint', authService.protect('admin'), (req: express.Reques
...
+ The *PassportAuth* is using _local_ *Passport.js* based strategy.
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc]: *PassportAuth* uses the _local_ *Passport.js* strategy.

**Review by jstaffor — CHANGES_REQUESTED** [review]

File: docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc (hunk 3 above)
Comment (witmicko) [reply docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc:9]: @wtrocki are you ok with this change?

File: docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc (hunk 1 above)
Comment (witmicko) [reply docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc]: will do

File: docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc (hunk 2 above)
Comment (witmicko) [reply docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc]: will do

File: docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc (hunk 4 above)
Comment (witmicko) [reply docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc]: done

**Review by jstaffor — APPROVED** [review]

**Pull request merged**